.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -30,6 +30,7 @@ body:
         - OpenTelemetry.Instrumentation.AWS
         - OpenTelemetry.Instrumentation.AWSLambda
         - OpenTelemetry.Instrumentation.Cassandra
+        - OpenTelemetry.Instrumentation.ConfluentKafka        
```

Comment:
Please also add in here: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/cd7e8d3cb6cb73c9dcb1398ac5914081d7578524/.github/workflows/prepare-release.yml#L22-L23

That list probably wasn't there when you started this work it is a newer thing. Sorry about that!

Reply:
done